client/my-sites/sidebar-unified/use-site-menu-items.js

```diff
@@ -32,12 +38,42 @@ const useSiteMenuItems = () => {
 	 * avoids a situation where the user might be left with an
 	 * empty menu.
 	 */
-	return (
-		menuItems ??
-		buildFallbackResponse( {
-			siteDomain,
-		} )
+
+	const shouldShowLinks = true;
+	const shouldShowTestimonials = useSelector( ( state ) =>
+		get( state.siteSettings.items, [ selectedSiteId, 'jetpack_testimonial' ], false )
+	);
+	const shouldShowPortfolio = useSelector( ( state ) =>
+		get( state.siteSettings.items, [ selectedSiteId, 'jetpack_portfolio' ], false )
+	);
+	const shouldShowWooCommerce = useSelector(
+		( state ) => !! getPluginOnSite( state, selectedSiteId, 'woocommerce' )?.active
 	);
+	/*
+	 * Header controlled by: current_theme_supports( 'custom-header' ) && current_user_can( 'customize' )
+	 * Background controlled by: current_theme_supports( 'custom-background' ) && current_user_can( 'customize' )
+	 * "What the theme supports" doesn't seem to be available in calypso most of the time?
+	 * Example theme w/ these options: "Dara"
+	 */
+	const shouldShowThemes = useSelector( ( state ) =>
+		canCurrentUser( state, selectedSiteId, 'edit_theme_options' )
+	);
+	const shouldShowApperanceHeaderAndBackground = true;
```

Comment:
Need to work on this one.

Reply:
Let's not worry about these. We're getting overly complicated for what is in essence a fallback.